README.md

```diff
@@ -1,4 +1,18 @@
 # Informal Audits
```

Comment:
Tagged in @andrey-kuprianov just for review on this part, in case he has ideas/preferences on how to improve this or how to present MBT.